module/applications/actor/npc-sheet.mjs

```diff
@@ -125,4 +126,15 @@ export default class ActorSheet5eNPC extends ActorSheet5e {
     // Parent ActorSheet update steps
     return super._updateObject(event, formData);
   }
+
+  /* -------------------------------------------- */
+  /*  Event Listeners and Handlers                */
+  /* -------------------------------------------- */
+
+  /** @override */
+  async _onDropSingleItem(itemData) {
+    const monsterFeature = (itemData.type === "feat") && !itemData.system.type.value && ("monster" in CONFIG.DND5E.featureTypes);
```

Comment:
This sort of automatic data population is typically handled in `Item5e#_preCreate`, that way it will apply to programmatic creation as well. Unless you specifically wanted to avoid that?

Reply:
I did not know of it at the time. It simplifies the PR greatly.